crates/wash-cli/src/dev.rs

```diff
@@ -1875,6 +1923,10 @@ pub async fn handle_command(
 
     // Watch FS for changes and listen for Ctrl + C in tandem
     eprintln!("👀 watching for file changes (press Ctrl+c to stop)...");
+    let _ = reload_rx.try_recv();
+    let _ = reload_rx.try_recv();
+    let _ = reload_rx.try_recv();
+    let _ = reload_rx.try_recv();
```

Comment:
What does this do?

Reply:
Ugh this was a leftover. I threw in a bunch to try and make sure we don't immediately rebuild but I think we need at most one

Reply:
I actually fixed this but I must not have force pushed.

Reply:
![i don't believe you](https://media1.tenor.com/m/z1NToU5JoJsAAAAC/i-dont-believe-you-whatever.gif)